packages/commonwealth/shared/analytics/types.ts

```diff
@@ -1,5 +1,5 @@
-import { CommunityType } from 'client/scripts/views/pages/create_community';
-import { ChainBase } from 'common-common/src/types';
+// import { CommunityType } from 'client/scripts/views/pages/create_community';
+// import { ChainBase } from 'common-common/src/types';
```

Comment:
delete. We can always get old code back from the repo. We don't want dead code lying around; unless the commented out code is absolutely crucial to the situation for some reason (rare case).

Reply:
Commented because I need to consult Alex before removing entirely.

Reply:
thats fine, we aren't using this too often anyways.